data/metadata/platemap/DEPENDENCIES1_A549.csv

```diff
@@ -0,0 +1,385 @@
+WellRow,WellCol,well_position,gene_name,pert_name,broad_sample,cell_line
```

Comment:
same comment for this file as the `barcode_platemap.csv` file.

Reply:
reasoning: https://github.com/WayScience/CytoPipe/pull/1#discussion_r845370067